010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
- _version | integer |           | not null | 0       | plain    |              |
- a        | text    |           |          |         | extended |              |
- b        | text    |           |          |         | extended |              |
- c        | bigint  |           |          |         | plain    |              |
- d        | boolean |           |          |         | plain    |              |
- e        | text    |           |          |         | extended |              |
- g        | text    |           |          |         | extended |              |
- h        | text[]  |           |          |         | extended |              |
-Indexes:
-"tbl_cd46p95cefoedabj77gg_a_pkey" PRIMARY KEY, btree (_id)
-    "tbl_cd46p95cefoedabj77gg_a_replica_id" UNIQUE, btree (_id, _version) REPLICA IDENTITY
-    "tbl_cd46p95cefoedabj77gg_a_xata_unique_b" UNIQUE CONSTRAINT, btree (b)
-Check constraints:
-    "tbl_cd46p95cefoedabj77gg_a_xata_multiple_length_h" CHECK (octet_length(array_to_string(h, ''::text)) < 65536)
-    "tbl_cd46p95cefoedabj77gg_a_xata_string_length_a" CHECK (length(a) <= 2048)
-    "tbl_cd46p95cefoedabj77gg_a_xata_string_length_b" CHECK (length(b) <= 2048)
-    "tbl_cd46p95cefoedabj77gg_a_xata_string_length_g" CHECK (length(g) <= 2048)
-    "tbl_cd46p95cefoedabj77gg_a_xata_text_length_e" CHECK (octet_length(e) < 204800)
-```
+A good example of this is the File Attachments faeture. While it is possible to store files as BLOBs in PostgreSQL, it would be expensive and inefficient to do so. Instead, we use S3 for file storage and the Cloudflare CDN for file delivery and image transformations.
 
-We also rely on PostgreSQL to enforce constraints (unique, not null), data size limitations, and referential integrity for the linked columns. Array types, like `multiple`, are represented using PostgreSQL array types.
+Similarly, while PostgreSQL has full-text search capabilities, we can provide a significantly better full-text search experience by using Elasticsearch. A blog post comparing the two approaches is available [here](https://xata.io/blog/postgres-full-text-search-postgres-vs-elasticsearch).
 
-Our data model contains rich types, which refer to data types that allow for more complex and structured data representations that can include a short “string” column (containing a name, or a label) and a “text” column (containing maybe a blog post in Markdown). This means we can have the same defaults for representing and indexing those columns in both PostgreSQL and Elasticsearch.
+The difference between Xata and a cloud provider that provides multiple DBaaS offerings is that in Xata the secondary data stores are **integrated**, meaning that the data is automatically replicated and kept in sync. Also, they are avaiable via the same SDKs and APIs and authorization works the same way.
 
-## Branches and zero-downtime migrations
+### Global control plane for databases and branches
```

Comment:
This section feels a bit complex / wordy for what you're trying to showcase. I really like the use cases presented, but it feels like it could be simplified. I'm not sure how quite yet, but can think about it a bit more.